src/actions/articleActions.js

```diff
@@ -62,16 +65,15 @@ export const fetchComments = article => dispatch => {
 
 export const fetchArticles = () => dispatch => {
   dispatch(getArticlesInitiated(true));
-  return axiosInstance
```

Comment:
Why did you remove the return statement?

Reply:
The return statement has been replaced.